docs/general/builtins/scalar-functions.rst

```diff
@@ -2718,6 +2718,58 @@ function supports arrays of any of the :ref:`primitive types
     SELECT 1 row in set (... sec)
 
 
+.. _scalar-array_position:
+
+``array_position(anycompatiblearray, anycompatible [, integer ] ) → integer``
+-----------------------------------------------------------------------------
+
+The ``array_position`` function returns the subscript of the first
+occurrence of the second argument in the ``array``, or ``NULL`` if it's not
+present. If the third argument is given, the search begins at that subscript.
+The ``array`` must be one-dimensional. Comparisons are done using
+``IS NOT DISTINCT FROM`` semantics, so it is possible to search for ``NULL``.
```

Comment:
CrateDB currently doesn't support the ``IS NOT DISTINCT FROM`` comparison statement. Thus we should phrase this differently to express than NULL values are supported.
Maybe something like:
"The function supports comparison of ``NULL`` values."

Reply:
Updated in documents about null support.